### ISSUE TITLE
Checking compiler version constraints

### ISSUE THREAD

**Issue by Pawe1** [Memnarch/Delphinus#56]
When user tries to install package with `package_compiler_min` / `package_compiler_max` / `compiler_min` / `compiler_max` values set that doesn't meet IDe version there should be at least warning message.

**Comment by Memnarch** [comment]
yup. STill haven't fixed that :/. Idea was to add filtering an simulate a given compilerversion when forcing it. (For example if lowest compiler-version is berlin and you use XE8, the script will run the Berlin-Nodes)

**Comment by rlebeau** [comment]
I thought the whole purpose of `(package_)compiler_...` was to prevent this kind of thing from happening?  If the `min/max` is set to 10.2, then the package shouldn't be made available to XE7 at all.  Am I wrong?

**Comment by Memnarch** [comment]
That is correct. Just had not added a filter to the UI (though it shows the versions it applies to). I should go through all my issues here and work on them again (and way to much to do otherwhise)
